src/test/resources/testSigningKey.p8

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
How this private key generated?

Reply:
It's a publicly shared pkcs8 sample private key.
Not a real key generated by ASC if that's what you're aiming at.

Reply:
Had to check 😁